### ISSUE TITLE
NO config information of MobilenetV2

### ISSUE THREAD

**Issue by wx19941204** [github-luffy/PFLD_68points_Pytorch#22]
The code has no config information of MobilenetV2:
`class MobileNetV2(nn.Module):

    def __init__(self, input_channels=3, num_of_channels=None, nums_class=136, activation=nn.ReLU6):
        super(MobileNetV2, self).__init__()
        assert num_of_channels is not None
        self.num_of_channels = num_of_channels
        self.conv1 = nn.Conv2d(input_channels, self.num_of_channels[0], kernel_size=3, stride=2, padding=1)
        self.bn1 = nn.BatchNorm2d(self.num_of_channels[0])

        self.depth_conv2 = nn.Conv2d(self.num_of_channels[0], self.num_of_channels[0], kernel_size=3, stride=1,
                                     padding=1, groups=self.num_of_channels[0])
        self.bn2 = nn.BatchNorm2d(self.num_of_channels[0])`
Can you provide the imformation?

**Comment by github-luffy** [comment]
> The code has no config information of MobilenetV2:
> `class MobileNetV2(nn.Module):
> 
> ```
> def __init__(self, input_channels=3, num_of_channels=None, nums_class=136, activation=nn.ReLU6):
>     super(MobileNetV2, self).__init__()
>     assert num_of_channels is not None
>     self.num_of_channels = num_of_channels
>     self.conv1 = nn.Conv2d(input_channels, self.num_of_channels[0], kernel_size=3, stride=2, padding=1)
>     self.bn1 = nn.BatchNorm2d(self.num_of_channels[0])
> 
>     self.depth_conv2 = nn.Conv2d(self.num_of_channels[0], self.num_of_channels[0], kernel_size=3, stride=1,
>                                  padding=1, groups=self.num_of_channels[0])
>     self.bn2 = nn.BatchNorm2d(self.num_of_channels[0])`
> ```
> 
> Can you provide the imformation?

![image](https://user-images.githubusercontent.com/21214262/101867694-c3343f80-3bb6-11eb-9366-0e46c5e3a56f.png)
mobilenetv2：[paper ](https://arxiv.org/abs/1801.04381)

**Comment by wx19941204** [comment]
I set num_of_channels as [64, 96, 160, 320, 640]

`
class MobileNetV2(nn.Module):

    def __init__(self, input_channels=3, num_of_channels=[64, 96, 160, 320, 640], nums_class=136, activation=nn.ReLU6):
        super(MobileNetV2, self).__init__()
        assert num_of_channels is not None
        self.num_of_channels = num_of_channels
        self.conv1 = nn.Conv2d(input_channels, self.num_of_channels[0], kernel_size=3, stride=2, padding=1)
        self.bn1 = nn.BatchNorm2d(self.num_of_channels[0])

        self.depth_conv2 = nn.Conv2d(self.num_of_channels[0], self.num_of_channels[0], kernel_size=3, stride=1,
                                     padding=1, groups=self.num_of_channels[0])
        self.bn2 = nn.BatchNorm2d(self.num_of_channels[0])

        self.stage0 = self.make_stage(self.num_of_channels[0], self.num_of_channels[0], stride=2, stage=0, times=5,
                                      expansion=2, activation=activation)`
But the net  perform much worse than Resnest50, so I wonder your set of num_of_channels

**Comment by github-luffy** [comment]
Please understand the meaning of each channel，Original data：num_of_channels = [64, 128, 16, 32, 128]，64 and 128 are only convolutional channels at the beginning，16 and 32 are the rising channel of LinearBottleneck，128 is the last channel number.

class LinearBottleneck(nn.Module):

    def __init__(self, input_channels, out_channels, expansion, stride=1, activation=nn.ReLU6):
        super(LinearBottleneck, self).__init__()
        self.expansion_channels = input_channels * expansion

        self.conv1 = nn.Conv2d(input_channels, self.expansion_channels, stride=1, kernel_size=1)
        self.bn1 = nn.BatchNorm2d(self.expansion_channels)

        self.depth_conv2 = nn.Conv2d(self.expansion_channels, self.expansion_channels, stride=stride, kernel_size=3,
                                     groups=self.expansion_channels, padding=1)
        self.bn2 = nn.BatchNorm2d(self.expansion_channels)

        self.conv3 = nn.Conv2d(self.expansion_channels, out_channels, stride=1, kernel_size=1)
        self.bn3 = nn.BatchNorm2d(out_channels)

        self.activation = activation(inplace=True)  # inplace=True
        self.stride = stride
        self.input_channels = input_channels
        self.out_channels = out_channels

**Comment by wx19941204** [comment]
thank you for your patience!!
i will read the code carefully.
